lib/opensusebasetest.pm

```diff
@@ -1050,7 +1051,10 @@ sub wait_boot_past_bootloader {
     # Starting with GNOME 40, upon login, the activities screen is open (assuming the
     # user will want to start something. For openQA, we simply press 'esc' to close
     # it again and really end up on the desktop
-    send_key('esc') if match_has_tag('gnome-activities');
+    if (match_has_tag('gnome-activities')) {
+        send_key 'esc';
+        check_screen 'generic-desktop', $check_interval;
```

Comment:
as you already noted: in case we close activities with opensuse-welcome open, we fail here now

Reply:
Needs some perl fu to filter out `gnome-activities` from `@tags`

Reply:
`@tags = grep {!/gnome-activites/} @tags;`